catalog/ui/catalog-ui-search/package.json

```diff
@@ -5,9 +5,10 @@
   "version": "0.0.1",
   "license": "LGPL-3.0",
   "scripts": {
-    "start": "grunt start",
+    "start": "grunt start --webpackConfig=./webpack/config/dev.js",
```

Comment:
(I had to add this here, as the dev server no longer worked after https://github.com/codice/ddf/pull/1820 was merged, since it was now expecting a webpackConfig to be passed)